docs/gettingstarted.rst

```diff
@@ -229,7 +229,59 @@ a string or integer. Below is an implementation of this design in the Python cod
+are defined per guild, and each guild can have multiple permissions. Here table that shows the JSON 
+structure of how choices are represented for a single permission:
+
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| **Field**   | **Type**                                   | **Description**                                                                                     |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| id          | int                                        | Snowflake value of type specified. Represents the target to apply permissions on.                   |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| type        | int                                        | An `ApplicationCommandPermissionType`_.                                                             |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+| permission  | boolean                                    | true to allow, false, to disallow.                                                                  |
++-------------+--------------------------------------------+-----------------------------------------------------------------------------------------------------+
+
+How the type parameter works is very simple. Discord has many ids to represent different things. As you can 
+set permissions to apply for User or Role, `ApplicationCommandPermissionType`_ is used. It's a number and
+following table shows the supported id types for permissions:
+
++-------------------+-----------+
+| **Name**          | **Value** |
++-------------------+-----------+
+| Role              | 1         |
++-------------------+-----------+
+| User              | 2         |
++-------------------+-----------+
+
+Now, let take a look simple example. The slash command decorator have a permissions parameter Where
+it takes in a dictionary. The key being the guild to apply permissions on, and value being the list
+of permissions to apply. For each permission, we can use the handy ``create_permission`` method to 
+build the permission json explain above.
+
+In this case, we are setting 2 permissions for guild with id of ``12345678``. Firstly, we are allowing
+role with id ``99999999`` and disallowing user with id ``88888888`` from running the slash command.
+
+.. code-block:: python
+
+  @slash.slash(name="test",
+              description="This is just a test command, nothing more.",
+              permissions={
+                12345678: [
+                  create_permission(99999999, 1, True),
```

Comment:
There is no mention of importing this. Is this meant to refer to the required line needed below?

`from discord_slash.utils.manage_commands import create_permission`